plugins/virtualenvwrapper/virtualenvwrapper.plugin.zsh

```diff
@@ -1,55 +1,35 @@
-virtualenvwrapper='virtualenvwrapper.sh'
-virtualenvwrapper_lazy='virtualenvwrapper_lazy.sh'
-
-if (( $+commands[$virtualenvwrapper_lazy] )); then
-  function {
-    setopt local_options
-    unsetopt equals
-    virtualenvwrapper=${${virtualenvwrapper_lazy}:c}
-    source ${${virtualenvwrapper_lazy}:c}
-    [[ -z "$WORKON_HOME" ]] && WORKON_HOME="$HOME/.virtualenvs"
-  }
-elif (( $+commands[$virtualenvwrapper] )); then
-  function {
-    setopt local_options
-    unsetopt equals
-    source ${${virtualenvwrapper}:c}
-  }
-elif [[ -f "/usr/local/bin/virtualenvwrapper.sh" ]]; then
-  function {
-    setopt local_options
-    unsetopt equals
-    virtualenvwrapper="/usr/local/bin/virtualenvwrapper.sh"
-    source "/usr/local/bin/virtualenvwrapper.sh"
-  }
-elif [[ -f "/usr/share/virtualenvwrapper/virtualenvwrapper.sh" ]]; then
-  function {
+function {
+    # The equals option must be unset in order for virtualenvwrapper's init
+    # script to load (see #3893). Note that recent versions of virtualenvwrapper
+    # no longer seem to have this issue, so possibly this is no longer needed
     setopt local_options
     unsetopt equals
```

Comment:
Sorry, it looks like [we can get rid of this](https://github.com/ohmyzsh/ohmyzsh/pull/8521/files#issuecomment-581049715).

Reply:
OK, great